confirm-password.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Confirm Password</title>
+		<link rel="stylesheet" href="styles.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	</head>
+	<body>
+		<section>
+			<h2>Enter Password</h2>
+			<div class="password-container">
+				<input type="password" name="password" id="password" class="">
+   			</div>
+      <br>
+      <h2>Confirm Your Password</h2>
+      <div class="password-container">
+        <input type="password" name="confirm" id="confirm" class="">
+				<i id="confirm-icon" class="fa fa-circle show-icon" class=""></i>
+        </div>
+		</section>
+		<script src="showHide.js"></script>
```

Comment:
I don't think that you need this script. it's only for show hide and we don't need it here.

Reply:
Ok! will fix that. thx!